sway/commands/exec.c

```diff
@@ -0,0 +1,16 @@
+#include <string.h>
+#include "sway/commands.h"
+#include "log.h"
+#include "stringop.h"
+
+struct cmd_results *cmd_exec(int argc, char **argv) {
+	// TODO: config
```

Comment:
Speaking of which, we should get that subsystem spun back up sooner rather than later.